designer/server/src/test/scala/pl/touk/nussknacker/ui/process/deployment/DeploymentServiceSpec.scala

```diff
@@ -359,7 +356,7 @@ class DeploymentServiceSpec extends AnyFunSuite with Matchers with PatientScalaF
 
     val processDetails = fetchingProcessRepository.fetchLatestProcessDetailsForProcessId[Unit](id).dbioActionValues.value
     processDetails.lastStateAction shouldBe None
-    processDetails.isNotDeployed shouldBe true
+    processDetails.lastAction shouldBe None
```

Comment:
Here and below also should be lastStateAction

Reply:
Here we expect both lastStateAction and lastAction to be `None`.